osf/models/user.py

```diff
@@ -1416,3 +1416,21 @@ class Meta:
 def create_bookmark_collection(sender, instance, created, **kwargs):
     if created:
         new_bookmark_collection(instance)
+
+
+@receiver(post_save, sender=OSFUser)
+def create_quickfiles_project(sender, instance, created, **kwargs):
+    if created:
+        new_quickfiles_project(instance)
+
+
+@receiver(post_save, sender=OSFUser)
+def update_quickfiles_project_title(sender, instance, created, **kwargs):
+    from website.project import get_quickfiles_project_title
+    from osf.models.quickfiles import QuickFiles
+
+    if not created:
+        quickfiles = QuickFiles.objects.filter(creator=instance).first()
+        if quickfiles:
+            quickfiles.title = get_quickfiles_project_title(instance)
+            quickfiles.save()
```

Comment:
It'd be nice if we could avoid doing this on every OSFUser.save call.

Could you instead do this `OSFUser.save` and check if the user's name has changed (in `dirty_fields`)?